sympy/matrices/matrices.py

```diff
@@ -2616,7 +2630,38 @@ def dot(self, b):
             mat = mat.reshape(1, n)
         if b.shape != (n, 1):
             b = b.reshape(n, 1)
-        # Now ``mat`` is a row vector and ``b`` is a column vector.
+
+        """ Retrive the calue of hermitian and conjugate_convention from kwargs.
+        If it so happens that only conjugate_convention is passed then automatically set
+        hermitian to True. If only hermitian is true but no conjugate_convention is not passed
+        then automatically set it to "maths"
+        """
+
+        hermitian = kwargs.get("hermitian")
+        conjugate_convention = kwargs.get("conjugate_convention")
+
+        if(conjugate_convention != None and hermitian == None):
+            hermitian = True
+        if(hermitian == True and conjugate_convention == None):
+            conjugate_convention = "maths"
+
+        if(hermitian == True):
+            return mat.inner_product(b, conjugate_convention)
+        elif(hermitian == False):
```

Comment:
Don't use parentheses with an if. You can also just write `if hermitian` or `elif not hermitian`. I recommend reading through [PEP 8](https://www.python.org/dev/peps/pep-0008/).

Reply:
I will go through PEP8 and correct the code.